assets/css/frontend-form/layout3.css

```
@@ -253,12 +263,21 @@
   color: #EDB867;
 }
 .wpuf-form-add.wpuf-form-layout3 ul.wpuf-form li.wpuf-submit input[type=submit] {
-  background: #8726d3;
-  border-color: #8726d3 #8726d3 #8726d3;
-  -moz-box-shadow: 0 1px 0 #8726d3;
-  -webkit-box-shadow: 0 1px 0 #8726d3;
-  box-shadow: 0 1px 0 #8726d3;
+  background: #8726D3;
+  border-color: #8726D3;
+  border-radius: 85px;
+  border: none !important;
```

Stylelint (17.14.0)

[error] Overridden property "border-color" by shorthand "border" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)